src/SAML2/LogoutRequest.php

```diff
@@ -276,7 +276,7 @@ public function setSessionIndex(string $sessionIndex = null): void
      */
     public function toUnsignedXML(): DOMElement
     {
-        if ($this->encryptedNameId === null && $this->nameId === null) {
+        if ($this->encryptedNameId === null && !isset($this->nameId)) {
```

Comment:
In this file as well. Was psalm complaining about the comparison because the phpdoc said it couldn't be `null`?

Reply:
Yes, I think it complains about a RedundantCondition when explicitly testing for null